### PR TITLE
Fix: Update formatters to support new severity values

### LINKED ISSUE #3182: [Feature] Update formatters to support new severity values
As part of #3036 (and follow up of #3070 and #3071) we need to update all formatters (except HTML) to support the new severity values.

We might want to tweak the output of some of these (like maybe summary?)

### PR DESCRIPTION
**Pull request by sarvaje**
Fix #3182

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [x] Added/Updated related tests.

### CODE REVIEW

File: packages/formatter-codeframe/src/formatter.ts [hunk 1]
```diff
@@ -11,20 +11,20 @@
  * ------------------------------------------------------------------------------
  */
 
-import chalk from 'chalk';
+import chalk, { Chalk } from 'chalk';
```
Comment (molant) [inline packages/formatter-codeframe/src/formatter.ts]: Is `chalk` still needed now that it's in `utils`?

Comment (sarvaje) [reply packages/formatter-codeframe/src/formatter.ts]: yes, we still use `chalk.cyan` to format the resource.

**Comment by sarvaje** [comment]
I'm working on the tests right now :)

**Pull request merged**